ext/libp2p/peer/behaviour.rs

```diff
@@ -64,6 +64,7 @@ pub enum RequestResponseMessage {
 
 /// The events emitted by a [`RequestResponse`] protocol.
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
This seems unrelated to Lassie upgrade. Do you know why we need to allow dead code? Did you perhaps accidentally upgrade the libp2p version too?

Reply:
Yes please, I already created the issue - see https://github.com/CheckerNetwork/zinnia/issues/676